utiltydb.h

```diff
@@ -0,0 +1,30 @@
+#ifndef UTILTYDB_H
+#define UTILTYDB_H
+
+#include <QtSql>
+#include <QSqlDatabase>
+#include <qmessagebox.h>
+#include "schedule.h"
+
+class UtilityDB
+{
+public:
+    UtilityDB();
+    bool isConnected();
+    bool isTableExists(QString tableName);
+    void clearTable(QString tableName);
+    void dropTable(QString tableName);
+    void createScheduleTable(QString tableName);
+    Schedule getScheduleByTableName(QString tableName);
+    void insertSchedultToTable(QString tableName, Schedule schedule);
+
+private:
+    QSqlDatabase db;
+    void establishConnection();
+    void initialSetup();
+    QString DB_DRIVER_NAME = "QSQLITE";
+    QString DB_FILE_PATH = "./../../../../PNU-Schedule/database/schedule.db";
```

Comment:
This relative path should be replaced with QDir::currentPath() and some “cd” commands  + name of our .db file. The most important is thing here: this path should be the same for everyone.

Reply:
What's the point of replacing? Now we have relative path (not absolute), so what will change after use of method? Only additional methods call to stack.

Reply:
OK, let me explain. Except QDir::currentPath() we can also use [this method](https://doc.qt.io/qt-5/qcoreapplication.html#applicationFilePath) because when our app will be deploying, we don't know the user's actual app path. In QT it's possible to manipulate with these paths easily.